app/views/shoppings/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<main>
+<div class="container">
+
+
+<div class="max-width flex-center column shopping">
+  <h2>Shopping List</h2>
+  <div class="row max-width detail">
+    <p>
+      Amount of food items to buy: <%= @shop_list.length %>
+    </p>
+    <p>
+      Total value of food neeeded: <%= @total_price %>
+    </p>
+  </div>
+
```

Comment:
- [ ] How about we try to match the wireframe for shopping lists, currently it is a bit different like the heading should be centered amount and total should be horizontally aligned 
![image](https://user-images.githubusercontent.com/97521557/235313973-97d55226-b1c4-416b-ab2b-99a0bddc55e0.png)

the wireframe
![image](https://user-images.githubusercontent.com/97521557/235314042-92de82e7-8529-40d0-a3bb-b550611013e5.png)